lib/hooks/meta/index.js

```diff
@@ -0,0 +1,42 @@
+const { get } = require('lodash');
+const db = require('@asl/schema');
+
+module.exports = settings => {
+  const { Roles } = db(settings.db);
+
+  return model => {
+    const type = get(model, 'data.model');
+    const data = get(model, 'data');
+
+    switch (type) {
+      case 'pil':
+        data.subject = get(data, 'data.profileId'); // pil holder
+        data.establishmentId = get(data, 'data.establishmentId');
```

Comment:
I don't think this will be sufficient. Those parameters won't be on all cases, only the ones we have now with a `create` action.

I think we need to load the data from the database based on the id provided (if an id is provided).